src/title/TitleGroup.js

```diff
@@ -29,10 +29,12 @@ class TitleGroup extends PureComponent<Props> {
 
   render() {
     const { recipients } = this.props;
+    const { width, height } = Dimensions.get('window');
+    const numOfAvatar = Math.ceil(Math.max(width, height) / (32 + 16)); // ImgSize: 32, margin: 16
```

Comment:
I would suggest you to not not sure `Dimensions.get('window');`, as it will give Dimensions of window, but if app is running in split screen, it might break.

Also this solution doesn't seems to be future proof. As in future if you increase margin or size, you have to reconfigure this numbers.

Reply:
I can assure you that this will not break on split-screen. :)
`Dimensions.get('window')` will either be equal or larger than the actual width we want. In the latter case, this will render more avatars which is fine.

> Also this solution doesn't seems to be future proof. As in future if you increase margin or size, you have to reconfigure this numbers.

If I put a comment at those values, will that be fine? I can't think of any other workaround.

Reply:
> will either be equal or larger than the actual width we want. In the latter case, this will render more avatars which is fine.

If it doesn't actually affect what gets displayed, then the simpler thing is just to not do the slice at all (i.e. leave this effectively at infinity.) That will also avoid the risk that it could break due to changes in the assumptions (like the size and margin of these images) that this calculation is relying on.